parser.go

```diff
@@ -77,9 +77,7 @@ func init() {
 }
 
 func parserPkg(pkgRealpath, pkgpath string) error {
-	rep := strings.NewReplacer("\\", "_", "/", "_", ".", "_")
-	commentFilename, _ = filepath.Rel(AppPath, pkgRealpath)
-	commentFilename = commentPrefix + rep.Replace(commentFilename) + ".go"
+	commentFilename = commentPrefix + strings.Join(strings.Split(pkgpath, "/")[1:], "_") + ".go"
```

Comment:
you need to consider windows, Mac and Linux version

Reply:
The `commentFilename` does not include application path, and `pkgpath` only includes slashes. So there is no need to concern the cross-platform problem.